ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1,4 +1,32 @@
 periodics:
+- agent: kubernetes
+  labels:
+    ci.openshift.io/role: infra
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  cluster: ci/api-build01-ci-devcluster-openshift-com:6443
+  cron: ""
+  decorate: true
+  interval: 1m
```

Comment:
1m? There is a good chance an old instance of the job will still be running by that time, shouldn't we at least set `max_concurrency: 1`?

Reply:
Done.